workload_mapper/mariadb.rb

```diff
@@ -0,0 +1,6 @@
+mariadb_config = "/etc/my.cnf"
+if system.runs_service?("mysql") && system.has_file?(mariadb_config)
+  identify "mariadb", "db"
```

Comment:
just for the record: we decided to keep it this way so we can rename the services depending on what's going on for example mongodb can be a `db` service for a workload but considered a `queue` in a workload which uses postgres as it's `db` service